collectors/test-results/jira-xray/src/main/java/com/capitalone/dashboard/core/json/TestRunJsonParser.java

```diff
@@ -99,6 +99,9 @@ public TestRun parse(JSONObject jsonObject) throws JSONException {
         if(jsonObject.get(KEY_STATUS).equals("PASS")){
             return TestRun.Status.PASS;
         }
+        if(jsonObject.get(KEY_STATUS).equals("BLOCKED")){
```

Comment:
Possible case of NPE, please use StringUtils

Reply:
reformatted to be null safe